lib/Federation/Proxy/TalkV1/ProxyRequest.php

```diff
@@ -40,7 +40,6 @@ public function overwrittenRemoteTalkHash(string $hash): string {
 				'config' => [
 					'chat' => [
 						'read-privacy',
-						'typing-privacy',
```

Comment:
it's still overwritten, just not with a hardcoded value anymore, but the value might not be what the host send, so we should keep that.
In fact we might want to add the value in here, so when you change your setting the ones of the room also change?

Reply:
> In fact we might want to add the value in here, so when you change your setting the ones of the room also change?

I think we should do this? Otherwise remote capabilities are not refreshed, when you change your typing privacy?

Reply:
yeah, now we could extend the integration test:
Make a second copy of https://github.com/nextcloud/spreed/blob/86e13f9233e968fd7770a21b3057b0564aae0644/tests/integration/features/bootstrap/FeatureContext.php#L3803-L3834 which calls https://github.com/nextcloud/spreed/blob/9c3a8012c2864c5fb1710a039b92e631a98e9cc8/appinfo/routes/routesRoomController.php#L112-L113 to check that the capability also changed in the federated room and your overwrite worked